examples/webpack/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Polaris: Webpack example</title>
```

Comment:
TIL, webpack is always lowercase 😕 

```suggestion
    <title>Polaris: webpack example</title>
```